velox/docs/functions/spark/json.rst

```diff
@@ -22,6 +22,8 @@ JSON Functions
 
 .. spark:function:: get_json_object(json, path) -> varchar
 
-    Extracts a json object from path::
+    Extracts a json object from ``path``. Returns NULL if it finds json string
```

Comment:
It would be nice to document what values for 'path' are supported? Maybe provide a link to some existing documentation for path?

Reply:
@mbasmanova, I didn't find official document for valid json path. Just documented the pattern and gave some examples.